resources/definitions/stream30dual_mk2.def.json

```diff
@@ -0,0 +1,52 @@
+{
+    "version": 2,
+    "name": "VOLUMIC STREAM30DUAL MK2",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "VOLUMIC",
+        "manufacturer": "VOLUMIC",
+        "file_formats": "text/x-gcode",
+        "icon": "volumic-icon",
+        "platform": "STREAM30PRO_platform.STL",
+        "machine_extruder_trains":{"0": "stream30dual_0","1": "stream30dual_1"},
+        "has_machine_quality": true,
+        "has_materials": true
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "VOLUMIC STREAM30DUAL MK2" },
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 300 },
```

Comment:
Similarly, the website says this printer has a width of 260mm, not 300: https://imprimante-3d-volumic.com/en/3d-printer-range-detail-stream-30-dual-mk2/

I'm assuming you did it correctly here, and the difference is single-extrusion volume vs. dual-extrusion. But if it's indeed a mistake, let me know!

Reply:
Yes, sorry. fixed to 265 (the good value, our website have a mistake too, it will be corrected asap, thank you ;)